src/locale/en-US.js

```diff
@@ -0,0 +1,17 @@
+// Local translations when used outside of sense
+export default {
+  'Object.OrgChart': 'Org chart', // Name of chart, to be added to sense when native
+  'Object.OrgChart.Description':
+    'Displays the relations of an organization or similarly structured data. Starts at the top and allows for navigation through the hierachy.', // Description of chart, to be added to sense when native
+  'Object.OrgChart.MaxData': 'The maximum data limit is reached. The tree may display incorrectly.',
+  'Object.OrgChart.DummyRoot': 'Root root',
```

Comment:
why is this needed?

Reply:
good question, intended for tooltip on the extra root nodes, but probably not needed. Also strangely written, i'll remove it.